import/controls/TextSelectionController.qml

```diff
@@ -186,31 +188,14 @@ MouseArea {
         selectionController: root
     }
 
-    Notification {
+    Notice {
         id: notification
         property bool published
 
-        function show() {
-            if (published) {
-                close()
-            } else {
-                publish()
-                published = true
-            }
-        }
-
-        function hide() {
-            if (published) {
-                close()
-            }
-            published = false
-        }
-
-        expireTimeout: 3000
-        icon: "icon-s-clipboard"
```

Comment:
No chance to have an icon for the in-app notification?

Reply:
Icons were not part of the design.